analytics/management/commands/populate_analytics_db.py

```diff
@@ -88,7 +88,16 @@ def handle(self, *args: Any, **options: Any) -> None:
                 role=UserProfile.ROLE_REALM_OWNER,
             )
         do_change_user_role(shylock, UserProfile.ROLE_REALM_OWNER, acting_user=None)
-        stream = Stream.objects.create(name="all", realm=realm, date_created=installation_time)
+
+        administrators_user_group = UserGroup.objects.get(
+            name=UserGroup.ADMINISTRATORS_GROUP_NAME, realm=realm, is_system_group=True
+        )
```

Comment:
It's possible that we'll end up with repeats of this code in enough code paths that it'll make sense to extract a 1-line function for this sort of thing, or implement a lookup in a dictionary object mapping settings to their default system groups, or something, but I'm not sure what that should look like yet, so we should merge this as-is.

Reply:
Yeah I think we'll want some sort of common registry of these defaults, so that `fix_streams_can_remove_subscribers_group_column` doesn't need to duplicate this setting value (which will surely break in the future at some point when we're changing some default and there's 30 group permissions settings).